tornado_opentracing/context_manager.py

```diff
@@ -0,0 +1,25 @@
+from tornado import version_info as tornado_version
+
+if tornado_version < (6, 0):
+    from opentracing.scope_managers.tornado import (
+        tracer_stack_context as trace_context
+    )
+else:
+    def trace_context():
```

Comment:
This isn't necessarily a tracing construct.  Maybe something like `tornado_context`?